script/migrate_to_mozfun

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
```

Comment:
Since this is going to be a short-lived script, I don't think it should live directly under `script`. Perhaps it should go under `script/legacy` to indicate that it's a short-lived script that isn't intended to be maintained long-term.